java/vector/src/main/java/org/apache/arrow/vector/compare/ApproxEqualsVisitor.java

```diff
@@ -121,9 +126,9 @@ protected boolean compareStructVectors(NonNullableStructVector left) {
       return false;
     }
 
+    ApproxEqualsVisitor visitor = new ApproxEqualsVisitor(floatEpsilon, doubleEpsilon);
```

Comment:
should we also do this for compareListVectors/compareFixedSizeLiseVectors?

Reply:
Sure. Thanks.

Reply:
Seems compareListVector/compareFixedSizeListVectors in this class does not change?

Reply:
Sorry. Fixed now.